### PR TITLE
feat(resource-status/Details): add Resource group chips to details

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This displays host or service (depending on the selected resource type) groups as chips in the details panel:

image

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

**Review by Thebarda — COMMENTED** [review]
host-group
LGTM with latest changes

**Comment by jdelpierre** [comment]
When I look at @Thebarda screenshot, would it be possible to put a small "margin/spacing" between the chips?

File: www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx [hunk 1]
```diff
@@ -217,6 +218,23 @@ const getDetailCardLines = ({
...
     },
+    {
+      title: labelGroups,
+      field: details.groups,
+      xs: 12,
+      getLines: (): Lines => [
+        {
+          key: 'groups',
+          line: (
+            <div>
```
Comment (jdelpierre) [inline www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx:229]: Maybe it's here to add class or style CSS for margin between chips

Comment (Thebarda) [reply www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx:229]: I think, it's better to override the `className` on the `Chip` component below

Comment (Thebarda) [reply www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx:229]: I think, it's better to override the `className` on the `Chip` component below

**Comment by bdauria** [comment]
@jdelpierre @Thebarda I added some spacing: 

Screenshot from 2021-01-05 11-01-30

**Review by jdelpierre — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**